common/src/test/kotlin/streams/utils/JSONUtilsTest.kt

```diff
@@ -111,6 +111,81 @@ class JSONUtilsTest {
         assertEquals(cdcData, fromString)
     }
 
+    @Test
+    fun `should convert wgs2D with height null to PointValue`() {
+        // given
+        val timestamp = System.currentTimeMillis()
+
+        val expected = org.neo4j.values.storable.PointValue.fromMap(mapOf("crs" to "wgs-84", "latitude" to 12.78, "longitude" to 56.7).toMapValue())
+
+        val cdcMap = mapOf<String, Any>(
+            "meta" to mapOf("timestamp" to timestamp,
+                "username" to "user",
+                "txId" to 1,
+                "txEventId" to 0,
+                "txEventsCount" to 1,
+                "operation" to OperationType.created),
+            "payload" to mapOf("id" to "0",
+                "before" to null,
+                "after" to NodeChange(properties = mapOf("location" to mapOf("crs" to "wgs-84", "latitude" to 12.78, "longitude" to 56.7, "height" to null)),
```

Comment:
is the "height" attribute required here? can we skip it?

Reply:
It's actually part of the test case to see if we're able to deserialize it without having any issues. The reason I did it this way is even though we prevented serializer from generating null values, there might still be some old events generating height = null value, so we decided to cover this case as well.

Reply:
ah yes, sorry, got confused about the inputs and outputs...
I think splitting the test in 2 as suggested below and a couple of renames and/or explicit `//when`/`//then` comments would help here

Reply:
👍